doc/releases/release-notes-3.5.rst

```diff
@@ -879,6 +879,13 @@ Libraries / Subsystems
 
   * Added support for CAN FD.
 
+* RTIO
+
+  * Added atomic completion counter fixing a race caught by unit tests
+  * Added a :c:macro:`RTIO_SQE_NO_RESPONSE` flag for submissions when no completion notification
```

Comment:
it would be nice to have the RTIO doc page include all relevant doxygen groups, since right now this won't render as  a link since the `rtio_sqe_flags` group is not included in the docs. Actually I think all those groups for flags should probably be simple `@name`d sections.

Reply:
Understood, I'd like to tackle improving and revamping the RTIO docs in the 3.6 time frame as well anyways so this will be part of that.

Reply:
Absolutely! Thanks :)